com.unity.render-pipelines.high-definition/Runtime/Material/SubsurfaceScattering/SubsurfaceScatteringManagerRT.cs

```diff
@@ -331,5 +337,138 @@ static void ExecuteCombineSubsurfaceScattering(CommandBuffer cmd, SSSCombinePara
                 PushFullScreenDebugTexture(hdCamera, cmd, diffuseBufferRT, FullScreenDebugMode.RayTracedSubSurface);
             }
         }
+
+        class TraceRTSSSPassData
+        {
+            public SSSRayTracingParameters parameters;
+            public TextureHandle depthStencilBuffer;
+            public TextureHandle normalBuffer;
+            public TextureHandle sssColor;
+            public TextureHandle intermediateBuffer0;
+            public TextureHandle intermediateBuffer1;
+            public TextureHandle intermediateBuffer2;
+            public TextureHandle intermediateBuffer3;
+            public TextureHandle directionBuffer;
+            public TextureHandle outputBuffer;
+        }
+
+        TextureHandle TraceRTSSS(RenderGraph renderGraph, HDCamera hdCamera, TextureHandle depthStencilBuffer, TextureHandle normalBuffer, TextureHandle sssColor, TextureHandle ssgiBuffer, TextureHandle diffuseLightingBuffer, TextureHandle colorBuffer)
+        {
+            using (var builder = renderGraph.AddRenderPass<TraceRTSSSPassData>("Composing the result of RTSSS", out var passData, ProfilingSampler.Get(HDProfileId.RaytracingSSSTrace)))
+            {
+                builder.EnableAsyncCompute(false);
+
+                // Grab the SSS params
+                var settings = hdCamera.volumeStack.GetComponent<SubSurfaceScattering>();
+                passData.parameters = PrepareSSSRayTracingParameters(hdCamera, settings);
+                passData.depthStencilBuffer = builder.UseDepthBuffer(depthStencilBuffer, DepthAccess.Read);
+                passData.normalBuffer = builder.ReadTexture(normalBuffer);
+                passData.sssColor = builder.ReadTexture(sssColor);
+                passData.intermediateBuffer0 = builder.WriteTexture(renderGraph.CreateTexture(new TextureDesc(Vector2.one, true, true)
```

Comment:
Seems like the intermediateBuffersX are only used in this pass. Shouldn't they use CreateTransientTexture?